.github/pull_request_template.md

```diff
@@ -36,3 +36,10 @@ Fixes #TODO
 ### Verification
 
 <!-- TODO: Say how you tested your changes. -->
+
+
+### Beyond this PR
```

Comment:
I would keep these sections as comments so that they don't appear in the text of the issues/PRs, and only when filing one. The `markdown` attribute in the issue forms is equivalent to a comment as it is ["not submitted"](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#keys)

Otherwise every PR and issue will also add a backlink to the reviewers issue which will make it hard to read actual comments